presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoDriverUri.java

```diff
@@ -265,6 +268,15 @@ public void setupClient(OkHttpClient.Builder builder)
                 }
                 builder.addInterceptor(tokenAuth(ACCESS_TOKEN.getValue(properties).get()));
             }
+            if (EXTERNAL_AUTHENTICATION.getValue(properties).orElse(false)) {
+                if (!useSecureConnection) {
+                    throw new SQLException("Authentication using external authorization requires SSL to be enabled");
```

Comment:
I think authentication, as the authorization part happens on the server during inspection of the token. Still I'm not strongly opinionated here.

Reply:
authentication. Why authorization? Is it because OAuth2 is using authorization term? From Presto perspective `EXTERNAL_AUTHENTICATION` is more generic, do not say much about OAuth2 as this is handled by Presto Server and the web browser.